apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -1006,7 +1046,7 @@ unpack_bridge_conf(Type, PackedConf, TopLevelConf) ->
 %%
 %% * The corresponding bridge v2 should exist
 %% * The connector for the bridge v2 should have exactly one channel
-is_valid_bridge_v1(BridgeV1Type, BridgeName) ->
+bridge_v1_is_valid(BridgeV1Type, BridgeName) ->
```

Comment:
I feel like this should return `false` for entirely new bridges created after this refactoring (which should not be supported by V1 API)?

Reply:
I think this is a design decision. I think it is fine new bridges are also supported in the legacy API. That would be more consistent in my opinion since internally there is no difference between refactored bridges and bridges created from scratch as action/connector. I think we can adjust this function according to the decision once a decision has been made.